Cannot find module 'lodash/object/pick'

Trying to run an app with v0.24.1 produces the following error: 
```
module.js:338
    throw err;
          ^
Error: Cannot find module 'lodash/object/pick'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/home/igor/Projects/getel-react/node_modules/react-bootstrap/lib/OverlayTrigger.js:32:25)
    at Module._compile (module.js:430:26)
    at Object.require.extensions.(anonymous function) [as .js] (/home/igor/Projects/getel-react/node_modules/node-jsx/index.js:29:12)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
```

--- a/package.json
+++ b/package.json
@@ -84,7 +84,6 @@
     "karma-webpack": "^1.5.0",
     "less": "^2.4.0",
     "less-loader": "^2.1.0",
-    "lodash": "^3.5.0",
     "marked": "^0.3.3",
     "mocha": "^2.2.1",
     "mt-changelog": "^0.6.1",
@@ -109,6 +108,7 @@
   },
   "dependencies": {
     "babel-runtime": "^5.1.10",
+    "lodash": "^3.5.0",
     "classnames": "^2.0.0"
   }
 }

